src/traces/sankey/render.js

```diff
@@ -218,22 +107,50 @@ function linkModel(uniqueKeys, d, l) {
     };
 }
 
-function nodeModel(uniqueKeys, d, n) {
+function linkPath() {
```

Comment:
This path generator for links used to lived in our fork of d3-sankey and was added in this commit:
https://github.com/plotly/d3-sankey/commit/80c33c31f414c3e263a8bd49e445c281d65ac6b0 . It doesn't need to live in there and could be moved back in here.

This and the fix for large node padding are the only improvements that we had over d3-sankey v0.4.2.

Reply:
To substantiate this last comment, you can review the [changes of our forked d3-sankey has compared to v0.4.2](https://github.com/d3/d3-sankey/compare/master...plotly:master#diff-968964796da46b424b314b4359ccde31).